ibm/service/kubernetes/resource_ibm_container_alb.go

```diff
@@ -187,7 +217,6 @@ func resourceIBMContainerALBUpdate(d *schema.ResourceData, meta interface{}) err
 
 	if d.HasChange("enable") {
 		enable := d.Get("enable").(bool)
-		disableDeployment := d.Get("disable_deployment").(bool)
 		albID := d.Id()
 		params := v1.ALBConfig{
```

Comment:
Similarly to the other cases, `params` is only needed if `enable == true`, should be moved there.